### PR TITLE
switch 

### PR DESCRIPTION
**Pull request by briarsychung**

### CODE REVIEW

File: cocnept-designs/v2/index.html [hunk 1]
```diff
@@ -13,15 +13,42 @@
...
                 <p>Mon 05/12/20</p>
                 <p>10:00 AM</p>
                 <p>$500</p>
                 <div class="stretch"></div>
-                <button>Computer</button>
-                <button>Office</button>
-                <button>Menu</button>
+                <button onclick="cambiar('game')">Computer</button>
+                <button onclick="cambiar('office')">Office</button>
+                <button onclick="cambiar('menu')">Menu</button>
```
Comment (SheepTester) [inline cocnept-designs/v2/index.html:51]: lol why Spanish

File: cocnept-designs/v2/styles.css [hunk 2]
```diff
@@ -1,5 +1,37 @@
+
```
Comment (SheepTester) [inline cocnept-designs/v2/styles.css:1]: Empty line at the start? Scary

**Review by SheepTester — COMMENTED** [review]
We probably should use classes more instead of IDs in the CSS for MAXIMUM REUSABILITY

File: cocnept-designs/v2/index.html (hunk 1 above)
Comment (briarsychung) [reply cocnept-designs/v2/index.html:51]: switch is a keyword lol
didn't see this comment till now oops

Comment (SheepTester) [reply cocnept-designs/v2/index.html:51]: lol why not `switchMenu` whatever I already changed it to `switchMenu`

Comment (briarsychung) [reply cocnept-designs/v2/index.html:51]: idk I always prefer one-word names

Comment (SheepTester) [reply cocnept-designs/v2/index.html:51]: oof

**Pull request merged**